### DISCUSSION TITLE
关于2.4G TX模块烧录和使用的问题

### DISCUSSION THREAD

**Discussion by zouzhe1** [ExpressLRS/ExpressLRS#2861] [Q&A]
I use **ESP32+byte E28 sx1280 DIY PCB TX** module
# Question 1:
When I burn the firmware, there is an option called **UARD_INVERTED**, and I don't know how to choose it.
[https://www.expresslrs.org/quick-start/firmware-options/?h=uart_inverted#other -options:~:text=%E5%8B%BF%E5%90%AF%E7%94%A8%E5%AE%83%E3%80%82-, UART_INVERTED,-%E8%BF%99%E4%BB%85%E9%80%82%E7%94%A8](https://www.expresslrs.org/quick-start/firmware-options/?h=uart_inverted#other-options:~:text=your%20chosen%20power.-,UART_INVERTED,-This%20only%20works)
I didn't explain it clearly here. I have two remote controls, one is **Flysky I6X** (the OpenTXI6X firmware has been flashed and TX has been soldered from the motherboard);
There is also a **WFLY ET16S** remote control, which is planned to be connected to the sport pin of the module through the high-frequency head interface.
So, how should I choose UARD_INVENTED for these two remote controls?
# Question 2:
I have reserved **I2C OLED** pins and ADC 5-way button pins on my PCB, but I don't know where to set them to work.

**Comment by pkendall64** [comment]
Internal modules should not have it enabled. External modules don't really need it now as we auto-detect the inversion for external modules.
